test-data/unit/fine-grained.test

```diff
@@ -5800,3 +5800,123 @@ class M(type):
 [out]
 ==
 a.py:2: error: Argument 1 to "f" of "M" has incompatible type "int"; expected "str"
+
+[case testAwaitAndAsyncDef-skip-cache]
```

Comment:
Why skip cache here and in some tests below?

Reply:
To avoid running the test cases twice -- the second run doesn't seem very useful.